tests/integration/test_active_learning.py

```diff
@@ -49,7 +49,7 @@
     [
         (50, EfficientGlobalOptimization[SearchSpace, SupportsPredictJoint](PredictiveVariance())),
         (
-            70,
+            80,
```

Comment:
I think this one is a bit stochastic, sometimes fails sometimes passes

Reply:
You may be right. It did pass for me just now, but I might investigate relaxing the test slightly to make it consistent.